src/controller.coffee

```diff
@@ -465,18 +465,13 @@ class Controller
     delete params.rpp
     delete params.query_counter
     delete params.page
-    # (@ecoslado) Hack to avoid the underscore's
-    # Array methods
-    if [].each?
-      filters = {}
-      for key, value of params.filters
-        filters[key] = []
-        value.each (elem) ->
-          filters[key].push elem
-
-      params.filters = filters
-
-    return "#{prefix}#{qs.stringify(params)}"
+    # serialization filter. no function is allowed inside querystring params
```

Comment:
@ecoslado , here's the thing. Finally I got a glimpse of the error, using wirwinzer.de html code, and indeed the serialization of the querystring parameters produced a hipyerlong string with lots of function definitions.

I opted for simply removing any function definition that may be in querystring parameters, I'm absolutely positive we don't want function definitions there. I think it's clean enough. Hope it's elegant, too. :-)